website/content/docs/internals/telemetry.mdx

```diff
@@ -150,13 +150,15 @@ These metrics cover measurement of token, identity, and lease operations, and co
 | `vault.expire.fetch-lease-times`                                                                | Time taken to fetch lease times                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | ms       | summary |
 | `vault.expire.fetch-lease-times-by-token`                                                       | Time taken to fetch lease times by token                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | ms       | summary |
 | `vault.expire.num_leases`                                                                       | Number of all leases which are eligible for eventual expiry                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | leases   | gauge   |
+| `vault.expire.num_irrevocable_leases`                                                           | Number of irrevocable leases which are present in the system                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | leases   | gauge   |
```

Comment:
```suggestion
| `vault.expire.num_irrevocable_leases`                                                           | Number of irrevocable leases that cannot be revoked automatically                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | leases   | gauge   |
```

this phrasing might be more clear? if you're ever in doubt about phrasing, Ivana is a great resource!

Reply:
ah nice! yeah I'm gonna use this wording, thanks!

Reply:
irrevocable is a bit redundant in this case, but probably nice for a user CTL/CMD-F-ing through the documentation

Reply:
Ah yeah I was thinking of getting rid of `irrevocable` here, making it `Number of leases that cannot be revoked automatically`, and since that is in the metric name itself, users should still be able to find it on searching i think.

Reply:
yeah true, sounds good!

Reply:
2558e12